prod-refractr.yml

```diff
@@ -344,6 +344,12 @@ refracts:
   # SE-1564
 - support.mozilla.org/en-US/kb/canary-domain-use-application-dnsnet:  use-application-dns.net
 
+  # SVCSE-1946
+- support.mozilla.org/en-US/kb/end-support-mozilla-hubs:
+    - hubs.mozilla.org
+    - hubs.mozilla.com
+    - hubs.mozilla.com/labs/
```

Comment:
I think you can remove the path here if you are redirecting the entire domain it would never get hit

```suggestion
```